ansible/roles/master/handlers/main.yml

```diff
@@ -14,12 +14,15 @@
 
 - name: restart apiserver
   service: name=kube-apiserver state=restarted
+  when: groups['masters']|length == 1
```

Comment:
I would prefer this syntax if it works b/c it's used elsewhere in the project:

groups['masters'][0]